client/src/store/modules/profile.js

```diff
@@ -3,19 +3,25 @@ export default {
     // Request to the database
     async uploadProfile (ctx) {
       const profileData = {
-        nickname: 'Elon Mask',
-        bio: 'My name is Elon and I like SoftServe company!',
+        id: 1,
```

Comment:
Why you still using hardcoded values? Please remove all mocks and start to work with real data.